eli5/formatters/features.py

```diff
@@ -1,14 +1,16 @@
 # -*- coding: utf-8 -*-
 
+import six
+
 
 class FormattedFeatureName(object):
     """ Feature name that does not need any additional escaping.
     """
     def __init__(self, value):
         self.value = value
 
-    def __str__(self):
-        return str(self.value)
+    def format(self):
+        return six.text_type(self.value)
```

Comment:
Fixed now. Didn't think about python 2 string for a while, sorry for the mess :)